python/pylibcudf/pylibcudf/libcudf/strings/convert/convert_ipv4.pxd

```diff
@@ -8,11 +8,11 @@ from pylibcudf.libcudf.column.column_view cimport column_view
 cdef extern from "cudf/strings/convert/convert_ipv4.hpp" namespace \
         "cudf::strings" nogil:
     cdef unique_ptr[column] ipv4_to_integers(
-        column_view input_col) except +
+        column_view input) except +
```

Comment:
I assume that you've been making these changes to match the C++. I'm in favor of them for consistency, so thank you. JFYI they don't impact the correctness of the generated code at all, so this kind of time is the only time they get fixed.